package.json

```diff
@@ -20,26 +20,21 @@
     "babel-preset"
   ],
   "dependencies": {
-    "@babel/plugin-proposal-async-generator-functions": "^7.0.0-beta.40",
-    "@babel/plugin-proposal-object-rest-spread": "^7.0.0-beta.40",
-    "@babel/plugin-transform-async-to-generator": "^7.0.0-beta.40",
-    "@babel/plugin-transform-destructuring": "^7.0.0-beta.40",
-    "@babel/plugin-transform-dotall-regex": "^7.0.0-beta.40",
-    "@babel/plugin-transform-exponentiation-operator": "^7.0.0-beta.40",
-    "@babel/plugin-transform-function-name": "^7.0.0-beta.40",
-    "@babel/plugin-transform-modules-commonjs": "^7.0.0-beta.40",
-    "@babel/plugin-transform-parameters": "^7.0.0-beta.40",
-    "@babel/plugin-transform-spread": "^7.0.0-beta.40",
-    "@babel/plugin-transform-sticky-regex": "^7.0.0-beta.40",
-    "@babel/plugin-transform-unicode-regex": "^7.0.0-beta.40"
+    "@babel/plugin-proposal-async-generator-functions": "7.0.0-beta.44",
+    "@babel/plugin-proposal-object-rest-spread": "7.0.0-beta.44",
+    "@babel/plugin-transform-async-to-generator": "7.0.0-beta.44",
+    "@babel/plugin-transform-dotall-regex": "7.0.0-beta.44",
+    "@babel/plugin-transform-exponentiation-operator": "7.0.0-beta.44",
+    "@babel/plugin-transform-modules-commonjs": "7.0.0-beta.44",
+    "@babel/plugin-transform-sticky-regex": "7.0.0-beta.44"
```

Comment:
The `y` flag is available in Node.js 6: http://node.green/#ES2015-syntax-RegExp--y--and--u--flags--y--flag